packages/validation/src/expressions.ts

```diff
@@ -0,0 +1,13 @@
+/* istanbul ignore file */
+
+// remember to update link patterns in the squidex schema
+
+export const urlExpression =
+  "^(?:http(s)?:\\/\\/)[\\w.-]+(?:\\.[\\w\\.-]+)+[\\w\\-\\._~:\\/?#%[\\]@!\\$&'\\(\\)\\*\\+,;=.]+$";
+
+export const emailExpression =
+  "^[a-zA-Z0-9.!#$%&’*+\\/=?^_`'{|}~-]+@[a-zA-Z0-9-]+(?:.[a-zA-Z0-9-]+)*$";
+
+export const telephoneNumberExpression =
+  '^\\(?\\d{3}\\)?[\\s.-]?\\d{3}[\\s.-]?\\d{3,4}$';
```

Comment:
did you learn them or got them from chatgtp?

Reply:
better - squidex

Reply:
They are the same as we use for validating a field at cms